src/eddymotion/model/base.py

```diff
@@ -87,13 +87,16 @@ class BaseModel:
         "_model",
         "_mask",
         "_S0",
+        "_gtab",
```

Comment:
This is very specific of DWI - happy to move forward with it, but we may end up increasing the inheritance chain to have a more general abstract interface.

(this also applies to `_S0`, which is just one line above and was here before the PR, and same for `_b_max` below)

Reply:
Done. See comments in commit df51602. Resolving.